package.json

```diff
@@ -1,46 +1,47 @@
 {
   "private": true,
-  "packageManager": "yarn@1.22.22",
+  "packageManager": "yarn@4.5.0",
   "repository": "https://github.com/comunica/comunica/",
   "workspaces": [
     "engines/*",
     "packages/*",
     "performance/*"
   ],
   "scripts": {
-    "test-changed": "lerna run test --since HEAD",
-    "build-changed": "lerna run build --since HEAD",
-    "test": "jest",
-    "test:ci": "jest --ci --maxWorkers=4 --coverage",
```

Comment:
Why was this removed?

Reply:
These scripts were renamed, and the CI test now uses the same script as the normal tests. I changed the normal test script to use `--runInBand` because it is surprisingly faster that way (easy to test locally, it took around 90 seconds without it, around 55 with it, if I remember correctly).

Reply:
We need to keep this, because jest does some things differently when run in a CI: https://jestjs.io/docs/cli#--ci

You probably have some caching going on, because `runInBand` takes almost 4 minutes on my system.

Reply:
It seems you were right, I ran all three versions locally to test, with the `--ci` flag, on this branch:
* `jest --selectProjects combined --ci --runInBand` ~60...70s
* `jest --selectProjects combined --ci --maxWorkers 4` ~30s
* `jest --selectProjects combined --ci` ~42s

I think I must have used different branches between the two or something.